arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1219,22 +1227,6 @@
 				<key>descriptionText</key>
 				<string>Uses Windows credentials to access services hosted on a portal secured with Integrated Windows Authentication (IWA).</string>
 			</dict>
-			<dict>
```

Comment:
This change removes the Authenticate with OAuth and Token Authentication samples from the UI. Please revert it.

Reply:
I'm slightly confused. From my understanding, I should add `<dict>` to line 1230, is that correct? I am unsure because if added, it doesn't seem to have an end tag.

Reply:
I apologize for the confusion; I was referring to this block of changes. In case you didn't know, additions in your branch that do not exist in the target branch appear in green and removals from your branch that exist in the target branch appear in red. This group of changes reflect lines that were somehow removed from your branch. If they were to be merged with this PR, those lines would no longer exist in `v.next`.

Does that make sense?

Reply:
Yes, that clarifies things, thanks!

Reply:
Thank you for adding the removed lines back in. However, the indentation of those lines has now changed (which is why GitHub is still showing them as modified). Please fix the indentation again. When you're done and you've pushed your changes, please look at the PR and make sure that the only changes to this file are the eight new lines you added for the new sample.